src/components/ont/runs/ONTFlowcell.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div
+      style="width: 200px"
+      class="px-2 rounded overflow-hidden shadow-lg"
+      :class="flowcell_bg_colour"
+    >
+      <div class="text-xl mb-2">{{ coordinate }}</div>
+      <traction-form-group
+        id="input-group-flowcell-id"
+        label="Flowcell ID:"
+        label-for="flowcell-id"
+        label-align="left"
+      >
+        <BFormInput
+          :id="'flowcell-id-' + position"
+          placeholder="Scan flowcell ID"
+          :value="flowcellId"
+          :formatter="formatter"
+          :state="flowcellIdValidation"
+          @input="setFlowcellId({ $event, position })"
+        ></BFormInput>
+        <!-- This will only be shown if the preceding input has an invalid state -->
+        <traction-invalid-feedback id="input-flowcell-id-feedback"
+          >Enter at valid Flowcell ID (3 letters then at least 3 numbers)</traction-invalid-feedback
+        >
+      </traction-form-group>
+      <traction-form-group
+        id="input-group-pool-id"
+        label="Pool Library Barcode:"
+        label-for="pool-id"
+        label-align="left"
+      >
+        <BFormInput
+          :id="'pool-id-' + position"
+          v-model="barcode"
+          :formatter="formatter"
+          :state="barcodeState"
+          debounce="500"
```

Comment:
Neat solution, is this in ms?

Reply:
Whats `ms`?

Reply:
sorry, ms (milliseconds), just checking how long the wait is

Reply:
Ah sorry! Yes its milliseconds. An example for "500" is [here](https://bootstrap-vue.org/docs/components/form-input#debounce-support)